lib/langParser.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+const iso639 = require('iso-639-1')
+
+fs.readdir(`${__dirname}/../static/i18n`, (err, items) => {
+  const availableLanguages = {
+    Languages: {},
+  }
+  items.forEach(function(value) {
+    const langName = value.substring(0, value.indexOf('.'))
+    const normalizedName = langName.split('-')[0]
+    const nativeName = iso639.getNativeName(normalizedName)
+    availableLanguages['Languages'][nativeName] = langName
+  })
+  fs.writeFileSync(
+    `${__dirname}/../src/js/languages.js`,
+    'export default ' +
```

Comment:
A `.json` file saves the trouble of having to `export` things.

Reply:
And how cena I import a JSON file?

Reply:
`import name from 'file.json'`

Reply:
I’ve tried without success

Reply:
try `require()` ?